### PR TITLE
Apiv2 Concept Map Bug Fix + TXT File Update

### PR DESCRIPTION
**Pull request by Mehul-Gandhi**
Fix the bug on the main branch that crashes the server with an import error. Add the helper function `getMaxPointsSoFar` in a separate file

Fix `/students/${selectedStudent}/progressquerystring` which crashed on the previous version. Change the formatting of the API request to be consistent with the formatting in other files.

Update the text file to contain data for the quest, midterm, midterm with code, postterm, and postterm with code, provided by Dan Garcia in the newest version of the spreadsheet.

Note: Concept Map is very big when all nodes are open. The user must zoom out.
image

Postterm Code entry:
image

### CODE REVIEW

**Comment by Mehul-Gandhi** [comment]
API test endpoint 
`http://localhost:3333/api/v2/students/midwaythere@berkeley.edu/progressquerystring`
Output: "44234310040421242244400443344040"
The length of this string is 32 since there are 32 total entries across quest, midterm no code, midterm code, postterm code, postterm no code.
image


For test@berkeley.edu:
`http://localhost:3333/api/v2/students/test@berkeley.edu/progressquerystring`
Output: "44444444444444444444444444444444"
image

File: api/lib/helper.mjs
Comment (Connor-Bernard) [inline api/lib/helper.mjs:1]: This file should have a more specific name

File: website/src/views/conceptMap.js [hunk 1]
```diff
@@ -34,8 +32,9 @@ export default function ConceptMap() {
         let mounted = true;
         if (mounted) {
             setLoading(true);
-            apiv2.get('/students/' + selectedStudent + '/progressquerystring').then((res) => {
-                setStudentMastery(res.data)
+            apiv2.get(`/students/${selectedStudent}/progressquerystring`).then((res) => {
+                //.substring(0, 6) for quest data only
+                setStudentMastery(res.data); //remove substring when we are ready for midterm and postterm
```
Comment (Connor-Bernard) [inline website/src/views/conceptMap.js:37]: Is there a reason why we are doing this and not just including it from the get go?  Also, if this is a task to be done later, it should be on the project board not as a comment (we won't see it / remember it if its a comment)

**Review by Connor-Bernard — CHANGES_REQUESTED** [review]

File: website/src/views/conceptMap.js (hunk 1 above)
Comment (Mehul-Gandhi) [reply website/src/views/conceptMap.js:37]: So what do you want to include on the launched version? All the exams or only a specific exam?

Comment (Connor-Bernard) [reply website/src/views/conceptMap.js:37]: All data currently on the sheet

Comment (Mehul-Gandhi) [reply website/src/views/conceptMap.js:37]: In that case, I can delete the comments because the application already does this.

**Review by Connor-Bernard — APPROVED** [review]
LGTM

**Pull request merged**